xonsh/completer.py

```diff
@@ -50,15 +132,19 @@ def startswithlow(x, start, startlow=None):
 
 
 def startswithnorm(x, start, startlow=None):
-    """True if x starts with a string s. Ignores its lowercase version, but
-    matches the API of startswithlow().
+    """True if x starts with a string s.
+
+    Ignores its lowercase version, but matches the API of startswithlow().
     """
+    _ = startlow
```

Comment:
It is a false conflict to pose readability against clean linting.  It is possible to have both, neither, and any combination in between.  I am happy to have the code lint cleanly, but not at the sacrifice of readability, and especially if that means performing extraneous operations like the assignment statements here. 

In this particular case, the function is meant to match a similar interface.  Thus I believe that the linter is being overly pedantic because it does not associate this function with its partner `startswithlow()`.  But because we know about this association the correct thing is probably to turn this linter error off. This happens in a comment (which is free in terms of execution) and not in an assignment to another unused variable name (which is just a convention and has a run time cost).